tests/Wordpress/CustomPostType/BookingTest.php

```diff
@@ -0,0 +1,242 @@
+<?php
+
+namespace CommonsBooking\Tests\Wordpress\CustomPostType;
+
+use CommonsBooking\Tests\Wordpress\CustomPostTypeTest;
+use CommonsBooking\Wordpress\CustomPostType\Booking;
+
+/**
+ * This class tests the form request for the frontend booking process
+ *
+ * TODO: Test the case, where one user creates an unconfirmed booking and an admin creates a booking for the same item, location and timeframe.
```

Comment:
Wenn ich das TODO richtig verstehe, testen wir es nicht da es noch nicht korrekt implementiert ist?

Reply:
Exakt! Aktuell ist es so, dass wenn ein Nutzender eine unconfirmed Buchung an  zb. Artikel A an Standort A von 10:00-12:00 erstellt, und gleichzeitig ein Admin für Artikel A an Standort A von 10:00-12:00 versucht eine Buchung zu erstellen, der Admin die Buchung von dem Nutzenden angezeigt bekommt.

Reply:
@hansmorb ich finde es wirklich klasse, wie detailliert ihr den Code testet und auch die Cornercases berücksichtigt. Den von dir beschriebenen Fall müssen wir aber denke ich nicht wirklich absichern. Die Eintrittswahrscheinlichkeit ist schon sehr gering, oder?

Reply:
@chriwen Das stimmt, ich hatte das auch nur in einem spezifischen Fall so! Hat auf jeden Fall keine Prio, deshalb habe ich auch nur diesen Kommentar geschrieben ;)